apps/studio/components/layouts/ProjectLayout/NavigationBar/NavigationBar.tsx

```diff
@@ -107,20 +107,20 @@ const NavigationBar = () => {
           if (!userDropdownOpen) snap.setNavigationPanelOpen(false)
         }}
       >
-        <ul className="flex flex-col gap-y-1 justify-start px-2">
-          {(!navLayoutV2 || !IS_PLATFORM) && (
+           {(!navLayoutV2 || !IS_PLATFORM) && (
             <Link
               href={IS_PLATFORM ? '/projects' : `/project/${projectRef}`}
-              className="mx-2 flex items-center h-[40px]"
+              className="mx-4 flex items-center h-[40px]"
               onClick={onCloseNavigationIconLink}
             >
               <img
                 alt="Supabase"
                 src={`${router.basePath}/img/supabase-logo.svg`}
-                className="absolute h-[40px] w-6 cursor-pointer rounded"
```

Comment:
Why do you need to add z-10 ?

Reply:
The z-10 ensures that the Supabase logo will appear above other elements on the page, such as backgrounds, images, or other content, that might otherwise overlap with it.